lib/dal/tests/integration_test/component/debug.rs

```diff
@@ -0,0 +1,68 @@
+use dal::component::debug::ComponentDebugView;
+use dal::prop::PropPath;
+use dal::{AttributeValue, Component, DalContext, Prop};
+use dal_test::test;
+use dal_test::test_harness::{commit_and_update_snapshot, create_component_for_schema_name};
+use pretty_assertions_sorted::assert_eq;
+
+#[test]
+async fn get_debug_view(ctx: &mut DalContext) {
```

Comment:
Let's leave some spaces and comments. This is difficult to read.

Usually, the convention we have to group things by `commit` when in doubt. Additionally, grouping by "preparing the environment" vs. performing mutations and assertions can be helpful.